examples/proc_FIR.py

```diff
@@ -11,10 +11,24 @@
 import pyspecdata as psd
 import sympy
 import matplotlib.pyplot as plt
+import numpy as np
 
-T1_list = []
 clock_correction = True
 plot_fit = True
+thisfile, exptype, post_proc, lookup = (
+    "240924_13p5mM_TEMPOL_ODNP_1.h5",
+    "ODNP_NMR_comp/ODNP",
+    "spincore_IR_v3",
+    prscr.lookup_table,
+)
+R1nodenames = [
+    "FIR_noPower",
+    "FIR_34dBm",
+]
```

Comment:
I moved these up b/c this is the stuff that you would change when using this script as an example.